data/schema.graphql

```diff
@@ -137,6 +136,9 @@ type Artist implements Node {
 
   # A string of the form "Nationality, Birthday (or Birthday-Deathday)"
   formatted_nationality_and_birthday: String
+
+  # A list of genes associated with an artist
+  genes: [Gene]
   gender: String
```

Comment:
Really big diff in this file, but it's a new one to me since I haven't used relay yet -- is this auto-generated?  Assuming since there are many changes that do not look tooltip related.

Reply:
Yes, that schema is synced from metaphysics, you need to do that when you add a new field to metaphysics to make it available in reaction `yarn sync-schema`